pkg/cluster/internal/create/createcontext.go

```diff
@@ -146,6 +146,12 @@ func (cc *Context) ProvisionNodes() (nodeList map[string]*nodes.Node, err error)
 			return nodeList, err
 		}
 
+		cc.Status.Start(fmt.Sprintf("[%s] Configuring proxy 🐋", configNode.Name))
+		if err := node.SetProxy(); err != nil {
+			// TODO: logging here
+			return nodeList, errors.Wrapf(err, "failed to set proxy for %s", configNode.Name)
```

Comment:
Also slightly iffy on this being a creation failure rather than a warning, but we can revisit this.